src/resources/echarts.ts

```diff
@@ -2,7 +2,7 @@
 import * as echarts from "echarts/core";
 
 // Import charts, all suffixed with Chart
-import { BarChart, LineChart, CustomChart } from "echarts/charts";
+import { BarChart, LineChart, CustomChart, SankeyChart } from "echarts/charts";
```

Comment:
Have we looked into dynamically loading parts of echarts? So `SankeyChart` will not be loaded for a history chart?

Reply:
This seems to work:

```ts
import { SankeyChart } from "echarts/charts";
import echarts from "./echarts";

// Register the required components
echarts.use([SankeyChart]);

export default echarts;
```

Reply:
I was testing the same thing. Just have to make the import async

Reply:
Turns out we don't even need `use`

Reply:
Alright I just pass the extra component to `ha-chart-base` and `use` it there. Otherwise we load `echarts` twice.

Reply:
It would not actually load it twice, as that is not possible, but your current solution is nice 👍